docs/pages/api-reference/operators/dedup.md

```diff
@@ -12,8 +12,14 @@ Operator to dedup keyless datasets (e.g. event streams).
 The list of columns to use for identifying duplicates. If not specified, all 
 the columns are used for identifying duplicates.
 
-Two rows of the input dataset are considered duplicates if and only if they have 
-the same values for the timestamp column and all the `by` columns.
+Two rows of the input dataset are considered duplicates if and only if they are in the same window
+and have the same values for the `by` columns.
```

Comment:
Let's also describe the case of window not specified.

If `window` is specified, two rows of the input dataset are considered duplicates when they are in the same window and have the same value for the `by` columns. If `window` is not specified, two rows are considered duplicates when they have the exact same values for the timestamp column and all the `by` columns.